docs/ci/demo

```diff
@@ -54,6 +54,13 @@ echo '' > ci/one.sh
 echo '' > ci/two
 echo '' > ci/three
 popper init --existing --stages=one,two,three ci
+
+mkdir ci2
+echo '' > ci/a.sh
+echo '' > ci/b.txt
+echo '' > ci/c.sh
+popper init --existing --infer-stages ci2
+
```

Comment:
Can we add a check that tests that only `a` and `c` are the stages?

Reply:
This is the only missing piece in order to merge this PR

Reply:
Yes yes, I am working on it. I'll notify when completed.

Reply:
no, what I mean is that, after `popper init --existing --infer-stages ci2` runs, the entry in the `.popper.yml` file for pipeline `c2` will only contain two stages (`a` and `c`). So you can `cat` the file and grep for those, and also `cat` to see that `b` **is not** part of the pipeline.